014-quarkus-panache-with-transactions-xa/src/main/java/io/quarkus/qe/rest/data/UserResource.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.qe.rest.data;
+
+import io.quarkus.hibernate.orm.rest.data.panache.PanacheRepositoryResource;
+import io.quarkus.panache.common.Page;
+import io.quarkus.panache.common.Sort;
+import io.quarkus.rest.data.panache.MethodProperties;
+import io.quarkus.rest.data.panache.ResourceProperties;
+
+import java.util.List;
+
+@ResourceProperties(exposed = true, hal = true, halCollectionName = "user_list", paged = true, path = "users")
```

Comment:
exposed = true
paged = true

Are redundant annotation properties, by default are set to true, so could be removed

Reply:
I did it on purpose, with the aim to see if those parameters may or may not change in the future, so then we aware of it

Reply:
the defaults are better to be removed as it better to test whether they could be changed it the future